vm-mock/vm.js

```diff
@@ -700,6 +705,15 @@ export default function createMockedABI(
           ERROR('not enough balance to transfer ' + amount + ' coins.');
         }
 
+        // We should not have this case, but someone could add a ledger.set without a balance
+        // Typescript could be used to avoid this kind of error
+        if (ledger.get(toAddress).balance === undefined) {
+          ledger.get(toAddress).balance = BigInt(0);
+        }
+        if (ledger.get(fromAddress).balance === undefined) {
+          ledger.get(fromAddress).balance = BigInt(0);
+        }
```

Comment:
the `ledger.get(fromAddress).balance` is already used previously in the function. Moreover if you set fromBalance to 0, the result will be a negative balance. so imho you should throw an error (the one wich is already throwed at line 705